api/src/main/java/com/epam/esm/ApplicationRunner.java

```diff
@@ -0,0 +1,12 @@
+package com.epam.esm;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/** Application launch point. */
+@SpringBootApplication
+public class ApplicationRunner {
+  public static void main(String[] args) {
```

Comment:
what indent quantity do u have? 
 it should be 4 spaces 

<img width="577" alt="Screenshot 2022-07-22 at 17 01 50" src="https://user-images.githubusercontent.com/39922259/180455598-021676bc-ce8f-48f7-93ce-6522a422950a.png">

Reply:
It's spotless ( googleJavaFormat() ) not me

Reply:
Fixed - switch to eclipse()